### DISCUSSION TITLE
After some time services just stop responding

### DISCUSSION THREAD

**Discussion by dibu28** [tteck/Proxmox#380] [Help Needed]
I've installed lxc containers using your scripts on proxmox.
And while everything is working fine.

I'm experiencing problem with Nginx Proxy Manager and UptimeKuma containers they just stop responding after a day or even some hours of work. And need to be manualy restarted. Any Idea why it can happen?

**Comment by tteck** [comment]
I actually use both of those, and have no issues after months of up time. 🤷🏻‍♂️
